<ui version="4.0" >
 <class>ChannelsDialog</class>
 <widget class="QWidget" name="ChannelsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>192</red>
        <green>192</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >:/images/channels.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolBox" >
      <property name="maximumSize" >
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex" >
       <number>2</number>
      </property>
      <widget class="QWidget" name="page" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>62</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Channels</string>
       </attribute>
       <attribute name="icon" >
        <iconset resource="images.qrc" >:/images/channels.png</iconset>
       </attribute>
       <attribute name="toolTip" >
        <string>Channels</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTreeWidget" name="treeWidget" >
          <property name="verticalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="verticalScrollMode" >
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <column>
           <property name="text" >
            <string/>
           </property>
          </column>
          <item>
           <property name="text" >
            <string>Indy</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/FileTypeAudio.png</iconset>
           </property>
           <item>
            <property name="text" >
             <string>Zoogie on Drums</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >:/images/graph-line.png</iconset>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Darkman Records</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >:/images/amsn.png</iconset>
            </property>
           </item>
          </item>
          <item>
           <property name="text" >
            <string>Video</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/FileTypeVideo.png</iconset>
           </property>
           <item>
            <property name="text" >
             <string>Interviews with a Coder</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >:/images/daway.png</iconset>
            </property>
           </item>
          </item>
          <item>
           <property name="text" >
            <string>Miscellaneous</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/informations_24x24.png</iconset>
           </property>
           <item>
            <property name="text" >
             <string>Test Channel 1</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text" >
            <string>RetroShare</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/rstray3.png</iconset>
           </property>
           <item>
            <property name="text" >
             <string>Retroshare's Demo Channel</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >:/images/rstray3.png</iconset>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>62</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>My Channels</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QListWidget" name="listWidget" >
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <item>
           <property name="text" >
            <string>Retroshare's Demo Channel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>160</width>
         <height>337</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>My Subscriptions</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QListWidget" name="listWidget_2" >
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text" >
            <string>Retroshare's Demo Channel</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Opensource Channel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; font-weight:600;">Messages&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QTreeWidget" name="msgWidget" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy" >
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="sortingEnabled" >
            <bool>true</bool>
           </property>
           <property name="columnCount" >
            <number>10</number>
           </property>
           <column>
            <property name="text" >
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Rank</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>From</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Title</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Msg</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Count</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Size</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Rs[Cert/Chan]IdSize</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>RsMsgId</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget" >
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; font-weight:600;">Message Text&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QTextBrowser" name="msgText" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget" >
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; font-weight:600;">Recommended Files&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="msgList" >
            <property name="contextMenuPolicy" >
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
